compose-v2/galaxy-server/Dockerfile

```diff
@@ -0,0 +1,138 @@
+    GALAXY_CONFIG_TOOL_PATH=$GALAXY_ROOT/tools \
+    GALAXY_VIRTUAL_ENV=$GALAXY_ROOT/.venv \
+    GALAXY_DATABASE_PATH=$GALAXY_ROOT/database
+
+ENV GALAXY_USER=galaxy \
+    GALAXY_GROUP=galaxy \
+    GALAXY_UID=1450 \
+    GALAXY_GID=1450 \
+    GALAXY_HOME=/home/galaxy
+
+ENV GALAXY_CONDA_PREFIX=$GALAXY_CONFIG_TOOL_DEPENDENCY_DIR/_conda \
+    MINICONDA_VERSION=4.7.10
+
+RUN groupadd -r $GALAXY_USER -g $GALAXY_GID \
+    && useradd -u $GALAXY_UID -r -g $GALAXY_USER -d $GALAXY_HOME -c "Galaxy user" --shell /bin/bash $GALAXY_USER \
+    && mkdir $GALAXY_HOME \
+    && chown -R $GALAXY_USER:$GALAXY_USER $GALAXY_HOME
+
+FROM build_base as build_miniconda
+COPY ./files/common_cleanup.sh /usr/bin/common_cleanup.sh
+# Install Miniconda
+RUN curl -s -L "https://repo.anaconda.com/miniconda/Miniconda2-${MINICONDA_VERSION}-Linux-x86_64.sh" > ~/miniconda.sh \
+    && /bin/bash ~/miniconda.sh -b -p $GALAXY_CONDA_PREFIX \
+    && rm ~/miniconda.sh \
+    && ln -s $GALAXY_CONDA_PREFIX/etc/profile.d/conda.sh /etc/profile.d/conda.sh \
+    && echo ". $GALAXY_CONDA_PREFIX/etc/profile.d/conda.sh" >> $GALAXY_HOME/.bashrc \
+    && echo "conda activate base" >> $GALAXY_HOME/.bashrc \
+    && export PATH=$GALAXY_CONDA_PREFIX/bin/:$PATH \
+    && conda config --add channels defaults \
+    && conda config --add channels bioconda \
+    && conda config --add channels conda-forge \
+    && conda install virtualenv pip ephemeris \
+    && conda clean --packages -t -i \
+    && /usr/bin/common_cleanup.sh
+
+FROM build_base as build_galaxy
+COPY ./files/common_cleanup.sh /usr/bin/common_cleanup.sh
+# Install Galaxy
+RUN apt update && apt install --no-install-recommends python-dev python-pip -y && rm -rf /var/lib/apt/lists/* \
+    && mkdir "${GALAXY_ROOT}" \
+    && curl -L -s $GALAXY_REPO/archive/$GALAXY_RELEASE.tar.gz | tar xzf - --strip-components=1 -C $GALAXY_ROOT \
+    && cd $GALAXY_ROOT \
+    && ./scripts/common_startup.sh \
+    && . $GALAXY_ROOT/.venv/bin/activate && pip install psycopg2 \
+    # Fix as cheetah was missing C library first
+    && pip uninstall cheetah -y \
+    && pip install cheetah \
+    && deactivate \
+    && rm -rf .ci .circleci .coveragerc .gitignore .travis.yml CITATION CODE_OF_CONDUCT.md CONTRIBUTING.md CONTRIBUTORS.md \
+              LICENSE.txt Makefile README.rst SECURITY_POLICY.md pytest.ini tox.ini \
+              client contrib doc test test-data .venv/lib/node_modules .venv/src/node-v10.13.0-linux-x64 \
+              .venv/include/node .venv/bin/node .venv/bin/nodeenv \
+    && /usr/bin/common_cleanup.sh
+
+FROM ubuntu:19.10 as final
+COPY ./files/common_cleanup.sh /usr/bin/common_cleanup.sh
+
+# Install HTCondor
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt update && apt install --no-install-recommends htcondor -y \
+    && rm /etc/condor/condor_config.local \
+    && find / -name '*.pyc' -delete | true \
+    && /usr/bin/common_cleanup.sh
+
+# Install CVMFS
+RUN apt update \
```

Comment:
Questions: Can we have the CMFS in a separate container that we just pull down and mount in if needed?